packages/earth-admin/src/services/places.tsx

```diff
@@ -91,3 +91,12 @@ export const handlePlaceForm = async (
     ? await addPlace(place, group)
     : await updatePlace(placeID, place, group);
 };
+
+export const getUniqueSlug = async (
```

Comment:
extract this in utils, it will be used for other routes too

Reply:
the function is coupled to the `PlacesAPIService` and that's the way all other methods from the services are. Extracting this function into utils would go against the norm we have in place right now.